stagemonitor/src/main/java/com/cerner/beadledom/stagemonitor/request/LogJsonRequestTraceReporter.java

```diff
@@ -60,10 +60,14 @@ public void reportRequestTrace(ReportArguments reportArguments) throws Exception
         CondensedCallStackElement condensedCallStackElement = getCondensedCallStack(requestTrace);
         logger.info(objectMapper
             .writeValueAsString(new RequestTraceWrapper(requestTrace, condensedCallStackElement)));
-      } catch (JsonProcessingException ok) {
-        // It's not valid JSON, this log should only contain JSON, so log to error logger instead
+      } catch (JsonProcessingException e) {
+        // Error writing call stack to JSON, this log should only contain JSON,
+        // so log to error logger instead.
         errorLogger
-            .error("Unable to serialize stack request trace to json. Request: {}", requestTrace);
+            .error(
+                "Unable to serialize stack request trace to json. Error: {} Request: {}",
+                e,
+                requestTrace);
```

Comment:
I think we could do that, but I'd rather take this as a first step and see if we need configuration options.